lib/ImageHelper.php

```diff
@@ -216,6 +202,24 @@ static function add_filters() {
 	}
 
 	//-- end of public methods --//
+
+	
+	/**
+	 * Checks if attachment is an image before deleting generated files
+	 *
+	 * @param  int  $post_id   an attachment post id
+	 *
+	 */
+	static function delete_generated_if_image( $post_id ) {
```

Comment:
Scrutinizer reports the following issue:

> It is generally recommended to explicitly declare the visibility for methods.

Since this function could be useful to use for others, I guess it’s ok introduce it as a `public` function.

```php
public static function delete_generated_if_image( $post_id ) {
```

And in that case, you’d better move it above the comment `//-- end of public methods --//`.